parsec/core/pki/exceptions.py

```diff
@@ -23,7 +23,13 @@
 
 
 class PkiEnrollmentError(Exception):
-    pass
+    @property
+    def message(self):
+        """All PkiEnrollmentError should have a message as first argument."""
```

Comment:
if it's a message, don't we want to do assert `isisnance(msg, str)` in the __init__ ?